app/assets/stylesheets/pages/challenge.sass

```diff
@@ -10,16 +10,16 @@
       .headline
         @extend .text-small
         line-height: $font-size-small
-    .descriptor-container
-      display: none
-      height: 0
 
 #challenge-nav
   @extend .text-center, .center-block, .text-small
   padding: 0px
   display: flex
   flex-direction: row
   justify-content: center
+  align-items: flex-start
+  @media (max-width: 768px)
```

Comment:
Let's use the variables defined in `bootstrap.sass` rather than hardcoding the breakpoints here. So this should / could be: `@media only screen and (max-width: $screen-sm)`